src/Sarif.Multitool/Rules/SARIF2009.UseConventionalSymbolicNames.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+using Microsoft.Json.Pointer;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool.Rules
+{
+    public class UseConventionalSymbolicNames : SarifValidationSkimmerBase
+    {
+        /// <summary>
+        /// SARIF2009
+        /// </summary>
+        public override string Id => RuleId.UseConventionalSymbolicNames;
+
+        /// <summary>
+        /// Adopt uniform naming conventions for the symbolic names that SARIF uses various contexts.
+        /// 
+        /// Many tools follow a conventional format for the 'reportingDescriptor.id' property: a short string identifying the tool concatenated with a numeric rule number,
+        /// for example, 'CS2001' for a diagnostic from the Roslyn C# compiler. For uniformity of experience across tools, we recommend this format.
+        /// 
+        /// Many tool use similar names for 'uriBaseId' symbols.We suggest 'REPOROOT' for the root of a repository, 'SRCROOT' for the root of the directory containing all source code, 'TESTROOT' for the root of the directory containing all test code(if your repository is organized in that way), and 'BINROOT' for the root of the directory containing build output(if your project places all build output in a common directory).
```

Comment:
>the [](start = 130, length = 3)

this comment line is too long and has spacing issues.

Reply:
just updated! Can you check again? Thank you

---
In reply to: [445553061](https://github.com/microsoft/sarif-sdk/pull/1938#discussion_r445553061) [](ancestors = 445553061)

Reply:
Better! My rule of thumb for this is to establish the right hand line length, typically from the first line. In your case, is is on line 23, the line ending in 'property:'. Then, I make sure and run all possible text up to this column. Basically attempt to right justify as far as possible. Following my convention, your comment could be more compact. The term 'for' at the beginning of line 25, for example, could be placed at the end of line 24 without exceeding the established line length. Make sense?

---
In reply to: [445569155](https://github.com/microsoft/sarif-sdk/pull/1938#discussion_r445569155) [](ancestors = 445569155,445553061)

Reply:
just updated! i think it is more justified and almost all lines are following the same length. Let me know what do you think!

---
In reply to: [445574287](https://github.com/microsoft/sarif-sdk/pull/1938#discussion_r445574287) [](ancestors = 445574287,445569155,445553061)